includes/variables.php

```diff
@@ -1432,17 +1432,16 @@ function tsml_load_config()
     ];
 
     // add 'TC', 'ONL', and languages to all programs
-    foreach ($tsml_programs as $key => $value) {
-        $tsml_programs[$key]['types'] = tsml_languages(array_merge([
-            'ONL' => __('Online Meeting', '12-step-meeting-list'),
-            'TC' => __('Location Temporarily Closed', '12-step-meeting-list'),
-        ], $tsml_programs[$key]['types']));
+    foreach ($tsml_programs as $tsml_program => $value) {
+        $tsml_programs[$tsml_program]['types']['ONL'] = __('Online Meeting', '12-step-meeting-list');
+        $tsml_programs[$tsml_program]['types']['TC'] = __('Location Temporarily Closed', '12-step-meeting-list');
+        $tsml_programs[$tsml_program]['types'] = tsml_languages($tsml_programs[$tsml_program]['types'], );
     }
 
     // remove 'TC' and 'ONL' from default flags if meeting finder is TSML UI
     if ($tsml_user_interface === 'tsml_ui') {
-        foreach ($tsml_programs as $key => $value) {
-            $tsml_programs[$key]['flags'] = array_diff($value['flags'], ['TC', 'ONL']);
+        foreach ($tsml_programs as $tsml_program => $value) {
```

Comment:
i think renaming this probably won't overwrite the `$tsml_program` variable we set on line 607, but only because it's not globalized on line 650 - seems dangerous though

Reply:
Oops. Thanks for the catch.